generator/internal/codegen_utils.cc

```diff
@@ -140,6 +140,26 @@ ProcessCommandLineArgs(std::string const& parameters) {
   return command_line_args;
 }
 
+std::string CopyrightLicenseFileHeader() {
+  static std::string const kHeader =  // clang-format off
```

Comment:
It would be better to declare this as `static auto constexpr kHeader`, because then it avoids potential static destruction issues w/ std::string.